### PR TITLE
Issue #738 FIXED 

### PR DESCRIPTION
**Pull request by SAMYAK99**
Screenshots for the change:
typo

### CODE REVIEW

**Comment by aggarwalpulkit596** [comment]
Did you check all the cases? because the more important thing is here that you show the correct information 
Different Cases - Trial, Expired , Not available on phone, etc kindly all these cases

**Comment by SAMYAK99** [comment]
I had checked most of the features , all  dialog boxes written  fine  . But  I  couldn't able to check the Expired  courses one as i didn't have one

File: app/src/main/res/layout/item_courses.xml [hunk 1]
```diff
@@ -94,7 +94,7 @@
             android:drawablePadding="10dp"
             android:drawableTint="@color/orangish"
             android:fontFamily="@font/gilroy_bold"
-            android:text="@string/buy_extension"
+            android:text="@string/buy_courses"
```
Comment (aggarwalpulkit596) [inline app/src/main/res/layout/item_courses.xml]: why change here ?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -243,11 +243,11 @@
     <string name="i_am_a">I am a</string>
     <string name="i_want_to_become">I want to become</string>
     <string name="choose_track">Choose Track</string>
-    <string name="purchase">"You need to buy this course to access this content"</string>
+    <string name="purchase">"You need to Buy this course to access this content"</string>
     <string name="ok">OK</string>
     <string name="cancel">Cancel</string>
     <string name="wait">"Your course will start on "</string>
-    <string name="expired">This section is unavailable as your course has been expired.Please buy an extension to watch your videos</string>
+    <string name="expired">You need to buy this course to access this content</string>
```
Comment (aggarwalpulkit596) [inline app/src/main/res/values/strings.xml]: Don't change existing values create new ones

**Review by aggarwalpulkit596 — CHANGES_REQUESTED** [review]

**Comment by aggarwalpulkit596** [comment]
@SAMYAK99  also PTAL
https://github.com/coding-blocks/CBOnlineApp/blob/9e4e85fe0f19c1dc18233effbb1b5142791ca1d2/app/src/main/java/com/codingblocks/cbonlineapp/mycourse/content/CourseContentFragment.kt#L376
As far as I understand this is more of the logic issue rather the typo error kindly check

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (SAMYAK99) [reply app/src/main/res/values/strings.xml]: Ok

**Comment by SAMYAK99** [comment]
What does it means   - ''viewModel.runStartEnd.first < System.currentTimeMillis() '' ?

**Comment by SAMYAK99** [comment]
I had fixed the buy now but   dialog box keeps loading !!!!!
Screenshot_1589634595

**Comment by aggarwalpulkit596** [comment]
> What does it means - ''viewModel.runStartEnd.first < System.currentTimeMillis() '' ?

Try to understand it by logging values

**Comment by aggarwalpulkit596** [comment]
> I had fixed the buy now but dialog box keeps loading !!!!!
> Screenshot_1589634595

You need to fix this one first then

**Pull request closed without merging**